social-network-api/src/main/java/fs/socialnetworkapi/utils/Universal.java

```diff
@@ -8,6 +8,11 @@
 public class Universal {
```

Comment:
We don't need this class. The main idea is that the user can be from different countries with different time zones. So, the backend won't know which timezone the user uses. It knows the frontend, so the frontend will calculate the time of the post, like (one hour ago, two weeks ago etc.)

Reply:
I'm not the author of this class

Reply:
But you are the owner of this project. Let's do it better :)